### PR TITLE
[OSCI] Adds new author rileyli

### PR DESCRIPTION
**Pull request by MadaniKK**
### Description
Added Riley Li as author for #osci2023
### Issues Resolved
Completed Task1 of OSCI_2023

### Check List
- [☑️ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/rileyli.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Riley Li
+short_name: Riley
+twitter:
```
Comment (joshuarrrr) [inline _authors/osci_2023/rileyli.markdown:4]: @MadaniKK Remove this line instead of leaving blank.

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
Remove the empty `twitter:` line. Otherwise looks good.

**Comment by dtaivpp** [comment]
@MadaniKK can you address Josh's requests? Thank you!

File: _authors/osci_2023/rileyli.markdown (hunk 1 above)
Comment (MadaniKK) [reply _authors/osci_2023/rileyli.markdown]: Done! sorry I didn't see the comment till now! Thank you for reviewing my PR!

**Comment by MadaniKK** [comment]
> @MadaniKK can you address Josh's requests? Thank you!
Done! Thank you!

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Review by dtaivpp — APPROVED** [review]
Looks great!

**Pull request merged**